CONTRIBUTE.md

```diff
@@ -0,0 +1,31 @@
+# Contribution Guidelines
+
+We welcome contributions towards code, documentation, testing, and translation. This guide should help explain the ways you can contribute and how to go about it. 
+
+## About the project
+If you are looking to contribute code, our project uses C++ with QT5. If you aren't comfortable making a code contribution there are lots of other ways to contribute, for example, improving documentation and testing across different platforms. The best approach is to either comment on an issue you are interested in contributing to, or create a new issue that describes the contribution you wish to make. 
+
+## How do I make a contribution?
+
+Never made an open source contribution before? Wondering how contributions work in the in our project? Here's a quick rundown!
+
+* Find an issue that you are interested in addressing or a feature that you would like to add. Create a new issue if necessary.
+* Fork the repository associated with the issue to your local GitHub organization. This means that you will have a copy of the repository under your-GitHub-username/tego-core.
+* Clone the repository to your local machine using git clone https://github.com/github-username/tego-core.git.
+* Create a new branch for your fix using git checkout -b branch-name-here.
+* Make the appropriate changes for the issue you are trying to address or the feature that you want to add.
+* If you are new to Git or GitHub see [https://guides.github.com/](https://guides.github.com/) for tutorials and guides.
+* Push your changes to your remote repository
+* Submit a pull request to the upstream tego-core repository
+* Provide a short description of the changes in the title of the pull request, reference the issue or bug number as well.
+* Provide more details of the changes in the pull description, in particular highlight any current weaknesses or outstanding problems. If your code has made changes to Crypto or protocol classes a detailed description of the changes and justification will be required. Please try to make pull requests for features separately - avoid bundling multiple issues or features into a single pull request. This makes code review easier and more manageable.
+* A maintainer will review the pull request and provide feedback where appropriate. Please be patient, our maintainers are volunteers and only have a limited amount of time to work on the project.
+* Make changes to the pull request if the reviewing maintainer recommends them.
```

Comment:
For the above section:
Since this is markdown, should we wrap commands with code tags?: \`code here\`.
i.e. git clone https://github.com/github-username/tego-core.git
becomes `git clone https://github.com/github-username/tego-core.git`

Reply:
Done see commit 6cd5c6b